frameworks/MLPlanWEKA/exec.py

```diff
@@ -0,0 +1,67 @@
+import logging
+import math
+import os
+
+from amlb.benchmark import TaskConfig
+from amlb.data import Dataset
+from amlb.datautils import reorder_dataset
+from amlb.results import NoResultError, save_predictions_to_file
+from amlb.utils import dir_of, path_from_split, run_cmd, split_path, Timer
+
+log = logging.getLogger(__name__)
+
+
+def run(dataset: Dataset, config: TaskConfig):
+    log.info("\n**** ML-Plan for WEKA ****\n")
+
+    # Mapping of benchmark metrics to Weka metrics
+    metrics_mapping = dict(
+        acc='ERRORRATE',
+        auc='AUC',
+        logloss='LOGLOSS',
+	f1='F1',
+	rmse="ROOT_MEAN_SQUARED_ERROR",
```

Comment:
Can you keep consistent indentation both here and below when defining `cmd_params`?

Reply:
sorry, of course I will make it consistent.